src/js/ripple/remote.js

```diff
@@ -519,16 +519,16 @@ Remote.prototype._handleMessage = function(message, server) {
     message = JSON.parse(message);
 
   /*eslint-disable no-empty*/
-  } catch (e) {
-    // TODO: why are we suppressing this?
-    console.error(e);
-  }
+  } catch (e) { }
```

Comment:
We don't specifically log that the parsing of the message fails. So we know the resulting message is not a valid message, but we don't know that it failed the JSON parsing. I don't really like silent catches